x-kiosk-imx8/Dockerfile

```diff
@@ -0,0 +1,230 @@
+RUN patch -p1 < 0001-ozone-add-va-api-support-to-wayland.patch
+
+COPY 0001-ozone-wayland-fix-re-initialization-of-WBMG.patch .
+RUN patch -p1 < 0001-ozone-wayland-fix-re-initialization-of-WBMG.patch
+
+COPY 0001-ozone-wayland-fixed-terminate-caused-by-binding-to-wrong-version.patch .
+RUN patch -p1 < 0001-ozone-wayland-fixed-terminate-caused-by-binding-to-wrong-version.patch
+
+#https://github.com/OSSystems/meta-browser/tree/master/meta-chromium/recipes-browser/chromium/files
+COPY 0007-Delete-compiler-options-not-available-in-release-ver.patch .
+RUN patch -p1 < 0007-Delete-compiler-options-not-available-in-release-ver.patch
+
+COPY 0014-ozone-wayland-don-t-build-xcb-for-pure-wayland-build.patch .
+RUN patch -p1 < 0014-ozone-wayland-don-t-build-xcb-for-pure-wayland-build.patch
+
+COPY 0015-IWYU-add-memory-for-std-unique_ptr-in-base-CommandLi.patch .
+RUN patch -p1 < 0015-IWYU-add-memory-for-std-unique_ptr-in-base-CommandLi.patch
+
+RUN autoninja -C out/Default/ chrome
+
+FROM ubuntu:20.04 
+
+LABEL maintainer="Raul Munoz <raul@foundries.io>"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get install -y vim curl wget libglib2.0-dev libnss3 libatk1.0-dev \
+                       libatk-bridge2.0-0 libjpeg-turbo8 libdrm-dev \
+                       libxkbcommon0 libasound2 libflac8 libxslt1.1 \
+                       fontconfig-config libwayland-cursor0 libwayland-client0 \
+                       libwayland-egl1 libxdamage1 libxfixes3 strace && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get -y update && apt-get install -y --no-install-recommends \
+    libwayland-client0 \
+    libwayland-server0 \
+    && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
+
+COPY libgal-imx_6.4.3.p1.4-aarch64-r0_arm64.deb /
+COPY libg2d1_6.4.3.p1.4-r0_arm64.deb /
+COPY libdrm2_2.4.102.imx-r0_arm64.deb /
+COPY imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb /
+
+RUN dpkg -i libgal-imx_6.4.3.p1.4-aarch64-r0_arm64.deb
+RUN dpkg -i libg2d1_6.4.3.p1.4-r0_arm64.deb
+RUN dpkg -i libdrm2_2.4.102.imx-r0_arm64.deb 
+RUN dpkg -i imx-gpu-viv_6.4.3.p1.4-aarch64-r0_arm64.deb
+
+RUN mkdir -p /usr/lib/chromium/
+
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/chrome /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/*.bin /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/icudtl.dat /usr/lib/chromium/
+COPY --from=build /chromium/chromium-96.0.4664.110/out/Default/chrome-wrapper /usr/lib/chromium/
+RUN ln -s /usr/lib/chromium/chrome-wrapper /usr/bin/chrome
+
+RUN mkdir -p /share/icons/hicolor/16x16/apps/
+RUN mkdir -p /share/icons/hicolor/22x22/apps/
+RUN mkdir -p /share/icons/hicolor/24x24/apps/
+RUN mkdir -p /share/icons/hicolor/32x32/apps/
+RUN mkdir -p /share/icons/hicolor/48x48/apps/
+RUN mkdir -p /share/icons/hicolor/64x64/apps/
+RUN mkdir -p /share/icons/hicolor/128x128/apps/
+RUN mkdir -p /share/icons/hicolor/256x256/apps/
```

Comment:
you've got a ton of RUN commands. Each one creates a docker layer. There are two ways you can improve this:
1) do a giant run in the Dockerfile like:
```
RUN \
    mkdir -p /share/icons/hicolor/16x16/apps/ && \
    mkdir -p /share/icons/hicolor/22x22/apps/ && \
    ....
```
2) create a build.sh type script and do `RUN build.sh`

Either approach will produce less layers and thus make it better to download

Reply:
Done!